schema.sql

```diff
@@ -34,4 +34,26 @@ create table owners(id SERIAL PRIMARY KEY,
 /*Add column owner_id which is a foreign key referencing the owners table */
   alter table animals
   add column owner_id int,
-  add constraint fk_owner foreign key(owner_id) references owners(id);
\ No newline at end of file
+  add constraint fk_owner foreign key(owner_id) references owners(id);
+
+  create table vets (
+  id INT GENERATED ALWAYS AS IDENTITY,
+  name varchar (300),
+  age int,
+  date_of_graduation date,
+  primary key (id));
+
+  create table specializations (
+  specialization_id int generated always as identity,
+  vet_id int,
+  species_id int,
+  vet_name varchar(300),
+  species_name varchar(200),
+  constraint pk_specialization primary key(specialization_id),
+  constraint fk_vet foreign key(vet_id) references vets(id),
+  constraint fk_species foreign key (species_id) references species(id));
```

Comment:
- You have done great so far :+1: But there are a few changes you need to make before moving to the next milestone. :+1: :wave: 

- [ ] Great job on creating the "specializations" join table! However, when creating a `many-to-many ` relationship join table, it's standard practice to use `IDs` instead of names for the relationship. This helps maintain data integrity and improves efficiency.
Instead of using "vet_name" and "species_name," consider using "vet_id" and "species_id" to establish the relationship between vets and species. I'll suggest you remove the `  vet_name varchar(300), and species_name varchar(200)` and work with the ids only. :wave: :smile: 

Also once you make this change, remember to update your data in `data.sql` and queries in `queries.sql` :wave: :airplane: